test/system/255-auto-update.bats

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests for automatically update images for containerized services
+#
+
+load helpers
+
+UNIT_DIR="/usr/lib/systemd/system"
+SNAME_FILE=$BATS_TMPDIR/services
+
+function setup() {
+    skip_if_remote "systemd tests are meaningless over remote"
+    skip_if_rootless
+
+    basic_setup
+}
+
+function teardown() {
+    while read line; do
+        if [[ "$line" =~ "podman-auto-update" ]]; then
+            echo "Stop timer: $line.timer"
+            systemctl stop $line.timer
+            systemctl disable $line.timer
+        else
+            systemctl stop $line
+        fi
+        rm -f $UNIT_DIR/$line.{service,timer}
+    done < $SNAME_FILE
+
+    rm -f $SNAME_FILE
+    basic_teardown
+}
+
+function generate_service() {
+    fake_img=$1
+    target_img=$2
+    cmd=$3
+    label=$4
+
+    cname=$(random_string)
```

Comment:
Adding a short static string can be a courtesy to the next person trying to debug a test failure (possibly you yourself):
```
   cname=c_$(random_string)
```
Then if you see `c_sdfsdfsdf` in test output you immediately know "this is the container name". If you just saw `sdfsdf`, it would be harder to know.

And the `echo` below, see my note on line 71